lambdas/initialise-ipv-session/src/main/java/uk/gov/di/ipv/core/initialiseipvsession/InitialiseIpvSessionHandler.java

```diff
@@ -238,6 +238,15 @@ public APIGatewayProxyResponseEvent handleRequest(
 
             auditService.sendAuditEvent(auditEvent);
 
+            if (isReverification) {
+                AuditEvent reverificationAuditEvent =
+                        AuditEvent.createWithoutDeviceInformation(
```

Comment:
could this be withDeviceInformation? generally if we have it (i.e. the event is user initiated) we should include it